src/components/gltf-model-plus.js

```diff
@@ -315,8 +317,17 @@ export const gltfModelPlus = {
                   }
 
                   for (let i = 0; i < clips.length; i++) {
-                    const action = self.mixer.clipAction(clips[i], node);
+                    const oneClip = clips[i];
+                    const action = self.mixer.clipAction(oneClip, node);
                     action.enabled = true;
+                    // If timeScale is set, use it.
+                    if (oneClip.componentProps.timeScale !== 1) {
+                      action.setEffectiveTimeScale(oneClip.componentProps.timeScale);
+                    }
+                    // If timeScale is set, use it.
+                    if (oneClip.componentProps.startOffset !== 0) {
+                      action.startAt(self.mixer.time - oneClip.componentProps.startOffset);
```

Comment:
I'm not sure about that one. Did you look at https://github.com/c-frame/aframe-extras/blob/master/src/loaders/animation-mixer.js#L126 to write this line? I'm not sure it's the same semantic here.
hubs is just setting action.time, see https://github.com/mozilla/hubs/blob/746413d43cba0a00f79f66b77fc9e559230c64ba/src/bit-systems/loop-animation.ts#L77
or https://github.com/mozilla/hubs/blob/746413d43cba0a00f79f66b77fc9e559230c64ba/src/components/loop-animation.js#L80

Reply:
Yes, I used the animation-mixer code as reference. I'll try if the Hubs approach to set the parameters directly also works.

Reply:
I changed the code to set the values as done in the Hubs code.